app/src/main/java/com/bulletin/extension/String_Compare.kt

```diff
@@ -0,0 +1,28 @@
+package com.bulletin.extension
+
+
+fun String.isNumeric(): Boolean {
```

Comment:
These methods can be combined with `StringExtensions.kt`

Reply:
will discuss